composer.json

```diff
@@ -27,7 +27,7 @@
         "keboola/input-mapping": "^18.9",
         "keboola/job-queue-internal-api-php-client": "^23.4",
         "keboola/object-encryptor": "^2.8",
-        "keboola/output-mapping": "24.23",
+        "keboola/output-mapping": "^24.25",
```

Comment:
Bude muset počkat (snad do pondělí) na https://keboola.atlassian.net/browse/PST-2051 a nebo revertnu `24.24` a vydám to jako `24.26`

To se přídávaly ty NT pro BQ. Když jsem to vyhazoval, tak jsem to jen downgradnul v runneru.

Reply:
jo to jo, ale v OM to je revertnutý viz
https://github.com/keboola/output-mapping/compare/24.24.0...24.25.0

Reply:
Vlastně máš pravdu, revertnul jsem to, jen jsem to nereleasoval, protože to nebylo nutný. Přídal jsem to teda do těch release notes :) https://github.com/keboola/output-mapping/releases/tag/24.25.0

Reply:
jo tvl to jsem tam taky mohl dát 🤦